### PR TITLE
Modified fetch_resources.sh to add interface to alternate internal network

### PR DESCRIPTION
**Pull request by priyankagupte15**
Modified fetch_resources.sh to add interface to alternate internal network

### CODE REVIEW

File: fetch_resources.sh [hunk 1]
```diff
@@ -227,7 +227,7 @@ function configure_tempest
         if [[ $TEST_IMAGE_NAME =~ "cirros" ]] ; then
             $OPENSTACK_CMD flavor create --ram 64 --disk 1 --vcpus 1 $TEST_IMAGE_NAME
         else
-            $OPENSTACK_CMD flavor create --ram 2048 --disk 20 --vcpus 2 $TEST_IMAGE_NAME
+            $OPENSTACK_CMD flavor create --ram 4096 --disk 20 --vcpus 2 $TEST_IMAGE_NAME
```
Comment (savitha-p) [inline fetch_resources.sh]: Revert this change. Looks like you need to pull latest changes from upstream.

File: fetch_resources.sh [hunk 2]
```diff
@@ -404,64 +455,26 @@ function configure_tempest
             network_id=`($OPENSTACK_CMD network list | grep test_internal_network | awk '$2 && $2 != "ID" {print $2}')`
```
Comment (nasharatm) [inline fetch_resources.sh:455]: This command fails on setups with different internal network. Ex => rhosp, we use 'trilio-internal-network'

**Review by nasharatm — CHANGES_REQUESTED** [review]

File: fetch_resources.sh (hunk 2 above)
Comment (priyankagupte15) [reply fetch_resources.sh:455]: This command will only run if no network is found on openstack. in the previous step it created internal network with test_internal_network name and then we run this command to get id.
For setup which has internal network created this step will not be executed and network id will be picked up irrespective of network name

Comment (nasharatm) [reply fetch_resources.sh:455]: looks good

**Review by nasharatm — APPROVED** [review]

**Review by savitha-p — CHANGES_REQUESTED** [review]

File: fetch_resources.sh (hunk 1 above)
Comment (priyankagupte15) [reply fetch_resources.sh]: reverted the change

**Review by savitha-p — APPROVED** [review]

**Pull request merged**